File: 프로그래머스 문제/level2/최댓값과 최솟값.py
def solution(s):
    answer = ''
    nums = s.split(' ')
    max_n = float('-inf')
    min_n = float('inf')
    for num in nums:
        if int(num) > max_n:
            max_n = int(num)
        if int(num) < min_n:
            min_n = int(num)
    answer = str(min_n) + ' ' + str(max_n)

    return answer

File: js연습/level2/최댓값과 최솟값.js
function solution(s) {
  var answer = '';
  let arr = s.split(' ')
  let a = 1000000000;
  let b = -1000000000;
  for (let i = 0; i < arr.length; i++) {
    if (Number(arr[i]) > b) {
      b = Number(arr[i])
    }
    if (Number(arr[i]) < a) {
      a = Number(arr[i])
    }
    console.log(a, b)
  }
  answer = String(a) + ' ' + String(b)
  return answer;
}